cirq-web/README.rst

```diff
@@ -1,20 +1,63 @@
-.. image:: https://raw.githubusercontent.com/quantumlib/Cirq/main/docs/images/Cirq_logo_color.png
-  :target: https://github.com/quantumlib/cirq
-  :alt: Cirq
-  :width: 500px
+.. |cirqlogo| image:: https://raw.githubusercontent.com/quantumlib/Cirq/refs/heads/main/docs/images/Cirq_logo_color.svg
+   :alt: Cirq logo
+   :target: https://github.com/quantumlib/cirq
+   :height: 100px
 
-`Cirq <https://quantumai.google/cirq>`__ is a Python library for writing, manipulating, and optimizing quantum
-circuits and running them against quantum computers and simulators.
+.. |cirq| replace:: Cirq
+.. _cirq: https://github.com/quantumlib/cirq
+
+.. |cirq-docs| replace:: Cirq documentation site
+.. _cirq-docs: https://quantumai.google/cirq
+
+.. |cirq-github| replace:: Cirq GitHub repository
+.. _cirq-github: https://github.com/quantumlib/Cirq
+
+.. |cirq-web| replace:: ``cirq-web``
+.. |cirq-core| replace:: ``cirq-core``
+
+.. class:: centered
+
+|cirqlogo|
+
+|cirq|_ is a Python package for writing, manipulating, and running `quantum
+circuits <https://en.wikipedia.org/wiki/Quantum_circuit>`__ on quantum
+computers and simulators. Cirq provides useful abstractions for dealing with
+today’s `noisy intermediate-scale quantum <https://arxiv.org/abs/1801.00862>`__
+(NISQ) computers, where the details of quantum hardware are vital to achieving
+state-of-the-art results. For more information about Cirq, please visit the
+|cirq-docs|_.
+
+This Python module is |cirq-web|, which allows users to take advantage of
+browser-based 3D visualization tools and features in Cirq. |cirq-web| also
+provides a development environment for contributors to create and add their own
+visualizations to the module.
+
+
+Installation
+------------
+
+|cirq-web| is currently in development, and therefore is only available in
```

Comment:
cirq-web is actually available as a "stable" package at https://pypi.org/project/cirq-web/ and it installs together with the cirq metapackage.  The installation info should be similar as for the other sub-packages.

Reply:
Oh, woops, that text needs to be corrected!

Reply:
Fixed in the Markdown version.